protocol/src/common/protocol.ts

```diff
@@ -607,6 +638,13 @@ export interface _ClientCapabilities {
 	 * Experimental client capabilities.
 	 */
 	experimental?: object;
+
+	/**
+	 * The client has support for file requests/notifications.
+	 *
+	 * Since 3.16.0
+	 */
+	files?: FileClientCapabilities
```

Comment:
And then we can name this fileOperations as well. Same on the server side.

Reply:
Done!

(Also pushed changes to the LSP spec PR to reflect the same)